### DISCUSSION TITLE
openclash启动失败

### DISCUSSION THREAD

**Discussion by zdiian** [vernesong/OpenClash#4010] [Q&A]
试过了很多次，把dev内核手动上传进去了如p1，但是还是启动失败p2
![23f2bd4b22dc4ade5118c80b6112de71](https://github.com/user-attachments/assets/1f884f11-c7a7-425c-8549-336c222c41bd)
![9803f5c804aaed34013c6ff025753967](https://github.com/user-attachments/assets/fa18d850-8039-4427-b2c0-3b9f4cb0cf2b)
因为我梯子的使用说明里说了要开启meta内核，所以我又去找了meta内核p3，但是我不知道为什么在dev旁显示meta，而meta哪一行没反应
![64a818a4fb6c53e9753f8e437f7894aa](https://github.com/user-attachments/assets/5289e143-dfc0-4d44-8f4e-ff8ee20af12c)
启动openclash失败后的日志
![608343241030d57fab9b7f5fea86d205](https://github.com/user-attachments/assets/727a7ed4-360f-451f-9188-ce229ab8cf92)
我每次失败总是会有这个下载失败，但是我并不知道这是什么东西，也没有找到教程，求助！！

**Comment by xiaoyangdkj** [comment]
在当前“版本更新”页面将“最新客户端”版本“下载到本地”，去opkg上传安装最新“luci-app-openclash”客户端版本.ipk后，返回到当前“版本更新”页面

选择“使用CDN一键检查更新”

**Reply by zdiian** [reply]
openclash可以正常启动了，但是openwrt pin不通，设备连软路由无法科学上网，但是可以正常上网，请问是什么情况
